custom-metrics-stackdriver-adapter/pkg/adapter/translator/query_builder.go

```diff
@@ -91,6 +91,74 @@ type Translator struct {
 	supportDistributions bool
 }
 
+// podsContainer is a helper struct to hold pods values
+type podsContainer struct {
```

Comment:
it's a container because it contains pod values, previously I named it as `podsHolder`, but I feel `podsContainer` is more descriptive, how do you think

Reply:
any suggestion on its naming (as well as for the node type)?

Reply:
in this case, how do you think of `podValues`? (since it holds two types of pod/nod values)